Guided Triage - Alerts.ipynb

```diff
@@ -86,7 +108,7 @@
     "pd.options.mode.chained_assignment = None\n",
     "from msticpy.nbtools.wsconfig import WorkspaceConfig\n",
     "from msticpy.sectools.ip_utils import convert_to_ip_entities\n",
-    "%env KQLMAGIC_LOAD_MODE=silent\n",
+    "\n",
```

Comment:
How would you feel about leaving this in to make the output cleaner but with a comment above it explaining what its doing and when to remove it?

Reply:
I left it in but moved it before TILookup(). If this has Azure Sentinel TI configured, this will trigger Kqlmagic load before the banner suppression env var. (see line 99)